src/v2/Components/Artwork/HoverDetails.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Pill } from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import { compact } from "lodash"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { HoverDetails_artwork } from "v2/__generated__/HoverDetails_artwork.graphql"
+
+interface HoverDetailsProps {
+  artwork: HoverDetails_artwork
+}
+
+const HoverDetails: FC<HoverDetailsProps> = ({ artwork }) => {
+  const labels = compact([
+    artwork.attributionClass?.name,
+    artwork.mediumType?.name,
+  ])
+  const pills = labels.map((label, index) => ({
+    id: `${artwork.internalID}-pill-${index}`,
```

Comment:
**question / non-blocking**: Is it important to use the artwork ID and pass it down to the NonClickablePill as its key or could we rely just on indexes instead? Do having the artwork ID encoded in the component key help with testing? I don't see it currently used for that purpose.

Reply:
> Is it important to use the artwork ID and pass it down to the NonClickablePill as its key or could we rely just on indexes instead?

It is better to always try to pass a unique value for key (this is considered good practice in [React](https://reactjs.org/docs/lists-and-keys.html#keys)). For this situation, we can also pass index as a key, but I decided to follow the recommended rules + this will help us to be safe and avoid problems that may arise if we use index for key (such problems are very difficult to debug)

Reply:
Gotcha. Yeah, seems like keys need to be unique among siblings and not globally but even then, it looks like React still recommends using IDs if available. Sounds good and thanks for clarifying! 👍🏼